cmd/rosa-support/delete/tag/cmd.go

```diff
@@ -0,0 +1,95 @@
+package tag
+
+import (
+	"os"
+
+	awsClient "github.com/openshift-online/ocm-common/pkg/aws/aws_client"
+	logger "github.com/openshift-online/ocm-common/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+var args struct {
+	region      string
+	resourceID  string
+	tagKey      string
+	tagValue    string
```

Comment:
i dont think the value should be something that is required during delete :)

Reply:
Due to possible issues of missed deletion:
https://docs.aws.amazon.com/cli/latest/reference/ec2/delete-tags.html#:~:text=If%20you%20specify%20the%20empty%20string%20as%20the%20tag%20value%2C%20the%20tag%20is%20deleted%20only%20if%20the%20tag%27s%20value%20is%20the%20empty%20string.%20The%20following%20delete%2Dtags%20example%20specifies%20the%20empty%20string%20as%20the%20tag%20value%20for%20the%20tag%20to%20delete.